dandi/upload.py

```diff
@@ -56,9 +57,13 @@ def upload(
             validation,
             dandiset_path,
             allow_any_path,
+            upload_dandiset_metadata,
             devel_debug,
         )
 
+    if upload_dandiset_metadata:
+        raise NotImplementedError("Not supported against original Girder based server")
```

Comment:
The error message should say what exactly it is that's not supported.

Reply:
good point, thanks